src/Adapter/Presenter/Cart/CartPresenter.php

```diff
@@ -475,6 +473,39 @@ function ($voucher) {
                 '',
         );
     }
+    
+    private function getShippingDisplayValue($cart, $shippingCost)
+    {
+        $hasFreeCarrier = 0;
+        $default_country = null;
+
+        if (isset(Context::getContext()->cookie->id_country)) {
+            $default_country = new Country(Context::getContext()->cookie->id_country);
```

Comment:
This could be improved but it's a bit outside of this PR scope. If we cannot find the country of the visitor because he has no cookies but he has landed on the EN version of the website or the GB version of the website, we could still check for the relevant carriers. But this is a bonus, not mandatory.